synapse/rest/client/v2_alpha/auth.py

```diff
@@ -221,5 +240,24 @@ def on_OPTIONS(self, _):
         return 200, {}
 
 
+class CasAuthTicketServlet(RestServlet):
+    PATTERNS = client_patterns(r"/auth/cas/ticket")
```

Comment:
This matches how the other CAS URLs are generated, but I believe someone mentioned we shouldn't be putting these under the `_matrix` prefix or something?